### ISSUE TITLE
"Plugin lib not found" message when applying a wallpaper

### ISSUE THREAD

**Issue by Giuszm** [catsout/wallpaper-engine-kde-plugin#266]
This happens when i select any of my wallpaper engine wallpapers

![Screenshot_20221218_175034](https://user-images.githubusercontent.com/109089108/208309888-3cbd8d92-d2ab-46e0-9a19-50fee19b4bda.png)

**Comment by hyperstown** [comment]
Have you followed instructions form [here](https://github.com/catsout/wallpaper-engine-kde-plugin#build-and-install)?
Last command:
```bash
# install lib
sudo make install
```

You can also verify if libs are installed in about section:

![image](https://user-images.githubusercontent.com/64496017/208493922-665a5b49-8853-4f27-a9cb-3ab573aa978d.png)

**Comment by Giuszm** [comment]
Yes i did follow the guide but know i don't remember if i skipped something important, should i redo "sudo make install"?
By the way this is my about section 
![Screenshot_20221219_201411](https://user-images.githubusercontent.com/109089108/208502565-b9e7890b-4c27-4c04-868c-b5ab6809d79d.png)

**Comment by hyperstown** [comment]
In my case after system update, plugin lib stopped working so I removed build folder from repo run `git pull` and redid steps from readme since `git submodule update --init`.

**Comment by hyperstown** [comment]
Probably `sudo make install` would suffice but I rebuild everything in case something changed.

**Comment by Giuszm** [comment]
I tried but still doesn't change anything, but now in the info tab of wallpaper engine plugin it says under Version: "warning: The lib version is inconsistent with the plugin version" 

![Screenshot_20221221_190330](https://user-images.githubusercontent.com/109089108/208974079-f59e2049-8774-4c3e-8e31-1e5cc422d45a.png)

**Comment by akippnn** [comment]
![image](https://user-images.githubusercontent.com/80670461/209862986-d232cbd2-efe9-482d-8cb8-c8880addb913.png)
Somehow, your plugin version is newer. You'd probably want them to match.

**Comment by Giuszm** [comment]
> ![image](https://user-images.githubusercontent.com/80670461/209862986-d232cbd2-efe9-482d-8cb8-c8880addb913.png) Somehow, your plugin version is newer. You'd probably want them to match.

how can i downgrade it?

**Comment by RabanalGsus** [comment]
I am having the same problem but can't my head around on how to downgrade it :/, if you get to a solution please let me know! Thanks in advance :D

**Comment by RabanalGsus** [comment]
> > ![image](https://user-images.githubusercontent.com/80670461/209862986-d232cbd2-efe9-482d-8cb8-c8880addb913.png) Somehow, your plugin version is newer. You'd probably want them to match.
> 
> how can i downgrade it?

So I succeeded working this problem around and getting a fix, for whatever reason when built from the command line I found this error to be consistent in my system (AMD 4800HS and GTX 1650ti, running Manjaro KDE) not mattering how many times I did it or undid it, but then it came to my mind the fact that Manjaro has the marvellous pamac manager from which I installed it, and surprise surprise it worked!! So it is not a definitive fix but it is a workaround if you have Manjaro ;)

**Comment by Jelgnum** [comment]
> So it is not a definitive fix but it is a workaround if you have Manjaro
So, you just used the AUR package then, which others using Arch based distros can use with their AUR wrapper of choice (yay, Paru, etc.) for anyone wondering how to do it in the future without using pamac or Manjaro.

**Comment by RabanalGsus** [comment]
Yep you are right, still wondering why building it from command line did not work as I was following the instructions. Maybe it is something to do with Arch based distros :/

**Comment by Jelgnum** [comment]
My best guess is that a dependency may have been missing, it does work on arch afaik because I have been using it for a while and have recompiled/rebuilt it in the past week.

**Comment by amuncx** [comment]
> 

what pkg did you install from aur,I install "aur/plasma5-wallpapers-wallpaper-engine" from aur,its version is 0.5.4,and the version of script is 0.5.5,but they all are inconsistent

**Comment by cesarhuret** [comment]
Getting the same error, tried installing via the AUR but doesn't seem to work. Also tried downloading the plugin from the Discover tab to install plugins, but the download keeps failing :sob:

**Comment by amuncx** [comment]
> Getting the same error, tried installing via the AUR but doesn't seem to work. Also tried downloading the plugin from the Discover tab to install plugins, but the download keeps failing sob

I have solved the issue,I deleted all of files and installed it with aur,youd better delete the file according to README

**Comment by cesarhuret** [comment]
> > Getting the same error, tried installing via the AUR but doesn't seem to work. Also tried downloading the plugin from the Discover tab to install plugins, but the download keeps failing sob
> 
> I have solved the issue,I deleted all of files and installed it with aur,youd better delete the file according to README

Ok, what were all the steps you specifically followed after uninstalling? Did you just run: `yay -S  plasma5-wallpapers-wallpaper-engine`?

**Comment by amuncx** [comment]
> > > Getting the same error, tried installing via the AUR but doesn't seem to work. Also tried downloading the plugin from the Discover tab to install plugins, but the download keeps failing sob
> > 
> > 
> > I have solved the issue,I deleted all of files and installed it with aur,youd better delete the file according to README
> 
> Ok, what were all the steps you specifically followed after uninstalling? Did you just run: `yay -S plasma5-wallpapers-wallpaper-engine`?

yes

**Comment by akippnn** [comment]
> > Use [waywallen](https://github.com/waywallen/waywallen).
> 
> Bro it's been 3 years i don't give a fuck anymore (not personal)

Why comment then? Just move on, I personally liked knowing there's an alternative.
